src/Client.php

```diff
@@ -384,4 +416,24 @@ protected function dispatchEvent($event): void
             $this->events_handler->__invoke($event);
         }
     }
+
+    /**
+     * Common query part for requests `user/repors` and `user/report_types`.
+     *
+     * @param array<string, string|int>|null $additional_params
+     *
+     * @return array<string, string|int>
+     */
+    private function getListQueryProperties(?array $additional_params = null): array
```

Comment:
Remove this method. DRY here is not required

Reply:
Method `getListQueryProperties` was removed 721dac95e68fe6cafe39dc9aa6d5706f26358d44